Content.Server/Body/Systems/RespiratorSystem.cs

```diff
@@ -251,7 +252,7 @@ private void OnCPRSuccess(CPRSuccessfulEvent ev)
             respirator.BreatheInCritCounter = respirator.BreatheInCritCounter + 3;
             respirator.CPRPlayingStream?.Stop();
 
-            if (!HasComp<MedicalTrainingComponent>(ev.Performer) && TryComp<PhysicsComponent>(ev.Patient, out var patientPhysics) && TryComp<PhysicsComponent>(ev.Performer, out var perfPhysics))
+            if (!HasComp<MedicalTrainingComponent>(ev.Performer) && !HasComp<HippocraticOathComponent>(ev.Performer) && TryComp<PhysicsComponent>(ev.Patient, out var patientPhysics) && TryComp<PhysicsComponent>(ev.Performer, out var perfPhysics))
```

Comment:
Perhaps make a binding to the staff affect the mass contest, or even the random roll instead, so that it has the effect of extremely good fortune, or a very gentle touch instead of simply replacing Medical Training. Make them feel better in some mystical way instead of like a trained doctor, of course being better if you are a doctor.